src/devices/GrovePi/Sensors/Relay.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.GrovePiDevice.Models;
+using System.Device.Gpio;
+
+namespace Iot.Device.GrovePiDevice.Sensors
+{
+    public class Relay : DigitalOutput
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="grovePi">The GoPiGo3 class</param>
+        /// <param name="port">The grove Port, need to be in the list of SupportedPorts</param>
+        public Relay(GrovePi grovePi, GrovePort port) : this(grovePi, port, false)
+        { }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="goPiGo">The GoPiGo3 class</param>
+        /// <param name="port">The grove Port, need to be in the list of SupportedPorts</param>
+        /// <param name="inverted">If inverted, the relay is on when output is low and off when output is high</param>
+        public Relay(GrovePi grovePi, GrovePort port, bool inverted) : base(grovePi, port)
+        {
+            IsInverted = inverted;
+        }
+
+        /// <summary>
+        /// Switch on the relay
+        /// </summary>
+        public void On()
+        {
+            Value = PinValue.Low;
+        }
+
+        /// <summary>
+        /// Switch off the relay
+        /// </summary>
+        public void Off()
+        {
+            Value = PinValue.High;
+        }
+
+        public bool IsInverted { get; set; }
```

Comment:
Comment

Reply:
Also, should set be public? Presumably flipping this is something you shouldn't need to do.